script.js

```diff
@@ -0,0 +1,53 @@
+const bookForm = document.querySelector('.book-form');
+const bookContainer = document.querySelector('.book-container');
+const titleInput = bookForm.title;
+const authorInput = bookForm.author;
+
+const books = JSON.parse(localStorage.getItem('books')) || [];
+
+const addBook = (title, author) => {
+  books.push({
+    title,
+    author,
+  });
+  localStorage.setItem('books', JSON.stringify(books));
+  return { title, author };
+};
+
+const createBook = ({
+  title, author,
+}, index) => {
+  const bookDiv = document.createElement('div');
+  const bookTitle = document.createElement('h2');
+  const bookAuthor = document.createElement('p');
+  const AremoveBtn = document.createElement('button');
+  const Ahr = document.createElement('hr');
+
+  bookTitle.innerText = `${title}`;
+  bookAuthor.innerText = `${author}`;
+  AremoveBtn.innerText = 'Remove';
+
+  bookDiv.append(bookTitle, bookAuthor, AremoveBtn, Ahr);
+  bookContainer.appendChild(bookDiv);
+
+  AremoveBtn.addEventListener('click', (e) => {
+    e.preventDefault();
+    books.splice(index, 1);
+    localStorage.setItem('books', JSON.stringify(books));
+    bookDiv.remove();
+  });
+};
+
```

Comment:
- [ ] As suggested previously when the remove button is clicked to remove a certain book, the book is not removed from the localStorage. Currently, one book is displayed in the local storage.
In the video below, one book is persistent in the local storage. Kindly fix it.

https://user-images.githubusercontent.com/70269507/176153902-319986d8-6e18-4575-b9ca-d091ff3686bb.mov